portugues/colabora/chora/chora.md

```diff
@@ -1,3 +1,8 @@
 Aos prantos, não lhe resta nada a não ser desistir de usar o app ou conviver com um bug inconveniente.
 Triste vida de alguém com tanto potencial e o desperdiça por falta de coragem...
-A menos que
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.